src/styles/variables.scss

```diff
@@ -0,0 +1 @@
+$catalog-size: calc((198px * 4) + (52px * 4));
```

Comment:
You have to use sass function for this statment. Checlist 13 [LAYOUT] - Use sass functions for dynamic max-width calculation, your catalog container should work fine for any number of cards.